recipe/meta.yaml

```diff
@@ -50,6 +50,11 @@ test:
     # Skip the import test on Linux as wheel file require a more recent
     # version of GLIBC++ than the VM used to build and test package.
     - tensorflow  # [not linux]
+  commands:
+    - tensorboard --help |grep USAGE         # [not win]
+    - freeze_graph --help |grep usage        # [not win]
+    - toco_from_protos --help |grep usage    # [not win]
+    - saved_model_cli --help |grep usage     # [not win]
```

Comment:
I'm not sure why `| grep usage` is required. That's explicitly testing for something we don't care about (the word usage in the help text) and seems redundant.

If the entrypoint isn't there the command will fail and hence the test will fail. If the entrypoint is there but the word *usage* isn't in the text, the test will also fail which isn't something we want. Maybe that would never happen in practice but it's still a redundant test and is preventing it being tested on Windows.

I'd be inclined to remove the `|grep usage` and test on all supported platforms.